src/main/resources/org/taktik/icure/asyncdao/impl/js/patient/By_hcparty_address.js

```diff
@@ -11,7 +11,7 @@ map = function (doc) {
 
 	var emit_patients_by_address = function (hcparty, doc, latin_map) {
 		doc.addresses.forEach(function (address) {
-			emit_normalized_substrings(hcparty, ((address.street || '') + (address.postalCode || '') + (address.city || '')).replace(new RegExp('\\s', 'g'), '').replace(new RegExp('\\W', 'g'), '').toLowerCase(), doc._id, latin_map);
+			emit_normalized_substrings(hcparty, ((address.street || '') + (address.postalCode || '') + (address.houseNumber || '') + (address.postboxNumber || '') + (address.city || '')).replace(new RegExp('\\s', 'g'), '').replace(new RegExp('\\W', 'g'), '').toLowerCase(), doc._id, latin_map);
```

Comment:
I do not think that is how it should be done SAM. I believe it is better to emit: ['postalCode', 'street'] and to filter the rest on the application server

Reply:
@aduchate are you suggesting here i'd better make a filter that accepts multiple parameters (street, number, ...) instead of 1 parameter that searches the concatenation of all address properties ?

Reply:
Yes.